src/components/LeftDrawerOrg/LeftDrawerOrg.tsx

```
@@ -93,149 +129,130 @@
+      </div>
 
-        {/* Organization Section */}
-        <div className={`${styles.organizationContainer} pe-3`}>
-          {loading ? (
+      {/* Organization Section */}
+      <div className={`${styles.organizationContainer} pe-3`}>
+        {loading ? (
+          <button
+            className={`${styles.profileContainer} shimmer`}
+            data-testid="orgBtn"
+          />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)